apps/user-portal/src/locales/en/views.ts

```diff
@@ -0,0 +1,865 @@
+                        description: "Couldn't update the approval",
+                        message: "Something went wrong"
+                    },
+                    success: {
+                        description: "Successfully updated the approval",
+                        message: "Update successful"
+                    }
+                }
+            }
+        },
+        changePassword: {
+            forms: {
+                passwordResetForm: {
+                    inputs: {
+                        confirmPassword: {
+                            label: "Confirm password",
+                            placeholder: "Enter the new password",
+                            validations: {
+                                empty: "Confirm password is a required field",
+                                mismatch: "The password confirmation doesn't match"
+                            }
+                        },
+                        currentPassword: {
+                            label: "Current password",
+                            placeholder: "Enter the current password",
+                            validations: {
+                                empty: "Current password is a required field",
+                                invalid: "Current password is invalid"
+                            }
+                        },
+                        newPassword: {
+                            label: "New password",
+                            placeholder: "Enter the new password",
+                            validations: {
+                                empty: "New password is a required field"
+                            }
+                        }
+                    },
+                    validations: {
+                        genericError: {
+                            description: "Something went wrong. Please try again",
+                            message: "Password reset error"
+                        },
+                        invalidCurrentPassword: {
+                            description: "The current password you entered appears to be invalid. Please try again",
+                            message: "Password reset error"
+                        },
+                        submitError: {
+                            description: "{{description}}",
+                            message: "Password reset error"
+                        },
+                        submitSuccess: {
+                            description: "The password has been changed successfully",
+                            message: "Password reset successful"
+                        }
+                    }
+                }
+            },
+            modals: {
+                confirmationModal: {
+                    heading: "Confirmation",
+                    message: "Changing the password will result in the termination of the current session. You will have to login with the newly changed password. Do you wish to continue?"
```

Comment:
Shall we fix the character length, since we have changed it to a typescript file?

Reply:
Length issues has been fixed